glean/schema/source/hs.angle

```diff
@@ -132,7 +132,7 @@ type Declaration =
     patSyn: PatSynDecl |
     class_: ClassDecl |
     method: MethDecl |
-    instance: InstanceDecl |
+    instance: InstDecl |
```

Comment:
This type change breaks Thrift compat. 

Given that this is a rename, can it be safely ignored?

Reply:
You can always ignore Thrift compat issues, but I guess in this case it will also break schema validation... let me check

Reply:
It's weird, schema validation succeeds, but I think it should fail. Could be a bug.

Reply:
Ah no, I know why: it's because the change only affects derived predicates. It's always safe to change derived predicates because there's no data translation needed.